package.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "module",
   "name": "del-website",
-  "version": "v5.5.0-Next",
+  "version": "v5.6.0-Next",
```

Comment:
Noting that before a new release is made on the GitHub repo, this **needs** to be changed to v5.6.0-Release.